### PR TITLE
sshuttle 0.73

### PR DESCRIPTION
**Pull request by tianx2**

### CODE REVIEW

**Comment by apjanke** [comment]
This looks good but I don't quite understand the wrapper script changes, so going to wait on another maintainer to vet this before merging.

File: Library/Formula/sshuttle.rb [hunk 1]
```diff
@@ -11,7 +11,10 @@ def install
     # Building the docs requires installing
     # markdown & BeautifulSoup Python modules
     # so we don't.
-    libexec.install Dir["*"]
-    bin.write_exec_script libexec/"src/sshuttle"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
```
Comment (mikemcquaid) [inline Library/Formula/sshuttle.rb]: Is this all written in Python now?

File: Library/Formula/sshuttle.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Sshuttle < Formula
   desc "Proxy server that works as a poor man's VPN"
   homepage "https://github.com/sshuttle/sshuttle"
-  url "https://github.com/sshuttle/sshuttle/archive/sshuttle-0.72.tar.gz"
-  sha256 "3ea217fa98e3887b0fb2229eb65b9548a4beea1947ed5949501e5ceec360ba19"
+  url "https://github.com/sshuttle/sshuttle/archive/v0.73.tar.gz"
+  sha256 "d78a6aa76f93ab69fd9685ce11b428a9b549c049a6c0104740c06c9b354a5ae3"
   head "https://github.com/sshuttle/sshuttle.git"
 
   bottle :unneeded
```
Comment (mikemcquaid) [inline Library/Formula/sshuttle.rb]: Should have a `depends_on :python if MacOS.version <= :snow_leopard` if it's using Python now.

File: Library/Formula/sshuttle.rb (hunk 1 above)
Comment (tianx2) [inline Library/Formula/sshuttle.rb]: `sshuttle` was always written in Python. It recently went through some code restructuring in the 0.73 release where using the original `homebrew` install no longer worked.

File: Library/Formula/sshuttle.rb (hunk 2 above)
Comment (tianx2) [inline Library/Formula/sshuttle.rb]: Added.

**Comment by tianx2** [comment]
FYI - there is no `test` block because I couldn't find a way to call without getting a non-zero exit code. Even calling with `--help` returns a non-zero exit code.

**Comment by mikemcquaid** [comment]
@tianx2 That's fine, you can use `shell_output` with the exit code as the second parameter.

**Comment by tianx2** [comment]
@mikemcquaid updated with test block.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew @tianx2; we appreciate it! :smile:

**Pull request closed without merging**